_includes/graphql/customer-input.md

```diff
@@ -8,7 +8,7 @@ Attribute |  Data Type | Description
 `email` | String | The customer's email address
 `firstname` | String | The customer's first name
 `gender` | Int | The customer's gender (Male - 1, Female - 2)
-`group_id` | Int | The group assigned to the user. Default values are 0 (Not logged in), 1 (General), 2 (Wholesale), and 3 (Retailer)
+`group_id` | Int | Deprecated. This attribute should not be exposed in GraphQL. The group assigned to the user. Default values are 0 (Not logged in), 1 (General), 2 (Wholesale), and 3 (Retailer)
```

Comment:
"should not be" implies there's a chance a user could see this attribute. Can we say: "Deprecated. Use <whatever> instead." or perhaps "This attribute has been deprecated because the contents are not applicable for GraphQL."